standards/RDA/ruleset.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruleset name="RDA">
+	<description>Reader's Digest Association WordPress Coding Standards</description>
+
+	<rule ref="WordPress-Core">
+		<!-- This rule is foolish and breaks many clean code uses -->
+		<exclude name="PEAR.Functions.FunctionCallSignature.Indent" />
+		<exclude name="WordPress.PHP.YodaConditions" />
```

Comment:
Removing this LGTM. There's an open discussion about this at https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1624 and at the core.